region/brinstar/red/Red Tower.json

```diff
@@ -2463,20 +2463,32 @@
     {
       "id": 113,
       "link": [9, 5],
-      "name": "Fast Climb with Space Jump",
+      "name": "Fast Climb with Space Jump, No Rippers",
       "requires": [
-        {"notable": "Climb with Space Jump"},
+        {"notable": "Fast Climb with Space Jump"},
         "SpaceJump",
         "canTrickyJump"
       ],
+      "wallJumpAvoid": true,
       "note": [
         "Climb the top of Red Tower with just Space Jump.",
-        "This can be done by dodging the Rippers and shooting the block as Samus is going upward, so she goes through the block as it breaks.",
-        "HiJump or killing the Rippers can help but are not required.",
-        "If the Rippers are killed, this can instead be done by shooting the block while falling right as it goes off screen,",
+        "This can be done by shooting the block as Samus is going upward, so she goes through the block as it breaks.",
+        "Alternatively, this can be done by shooting the block while falling right as it goes off screen,",
         "then using well-timed short space jumps to go up as rapidly as possible before the block respawns."
       ]
     },
+    {
+      "link": [9, 5],
+      "name": "Space Jump Climb Between the Rippers",
+      "requires": [
+        "canPreciseSpaceJump",
+        "canTrickyDodgeEnemies"
+      ],
```

Comment:
just looked at the video, this looks much harder, so this would push it up to from VH to expert, and the expert variant would not be notable

Reply:
Practiced this and it's only the last Ripper that's annoying. I found that space jumping against one wall was easier than weaving back and forth, like the video.